hbase-client/src/main/java/org/apache/hadoop/hbase/exceptions/ClientExceptionsUtil.java

```diff
@@ -53,25 +52,22 @@
 
   private ClientExceptionsUtil() {}
 
-  public static boolean isMetaClearingException(Throwable cur) {
-    cur = findException(cur);
+  public static boolean isMetaClearingException(Throwable t) {
+    t = unwrapRemoteException(t);
 
-    if (cur == null) {
+    if (t == null) {
       return true;
     }
-    return !isSpecialException(cur) || (cur instanceof RegionMovedException)
-        || cur instanceof NotServingRegionException;
+    return !regionDefinitelyOnTheRegionServerException(t);
   }
 
-  public static boolean isSpecialException(Throwable cur) {
```

Comment:
I think this method is used elsewhere? Not only for clearing meta. Is it safe to change the implementation?

Reply:
I grep the whole project and can't find any other places where the function is called. Maybe removed earlier? Or could you remember in which module?